proto/net.proto

```diff
@@ -106,6 +106,57 @@ message Weights {
     optional Layer ln2_betas = 6;
   }
 
+  message PolicyHead {
+    // Policy head
```

Comment:
Sorry for nitpicking, but this comment is not really needed. :)

Reply:
Removed.